include/psp2common/ctrl.h

```diff
@@ -0,0 +1,150 @@
+/**
+ * \usergroup{SceCtrl}
+ * \usage{psp2common/ctrl.h}
+ */
+
+
+#ifndef _PSP2COMMON_CTRL_H_
+#define _PSP2COMMON_CTRL_H_
+
+#include <vitasdk/build_utils.h>
+#include <psp2common/types.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef enum SceCtrlErrorCode {
+	SCE_CTRL_ERROR_INVALID_ARG   = 0x80340001,
+	SCE_CTRL_ERROR_PRIV_REQUIRED = 0x80340002,
+	SCE_CTRL_ERROR_NO_DEVICE     = 0x80340020,
+	SCE_CTRL_ERROR_NOT_SUPPORTED = 0x80340021,
+	SCE_CTRL_ERROR_INVALID_MODE  = 0x80340022,
+	SCE_CTRL_ERROR_FATAL         = 0x803400FF
+} SceCtrlErrorCode;
```

Comment:
(out of subject)
imo, now, we can check the size of enums with `VITASDK_BUILD_ASSERT_EQ` :)